src/o365/spo/commands/orgassetslibrary/orgassetslibrary-remove.ts

```diff
@@ -0,0 +1,137 @@
+        };
+
+        return request.post(requestOptions);
+      })
+      .then((res: string): void => {
+        const json: ClientSvcResponse = JSON.parse(res);
+          const response: ClientSvcResponseContents = json[0];
+          if (response.ErrorInfo) {
+            cb(new CommandError(response.ErrorInfo.ErrorMessage));
+            return;
+          }
+          else {
+            cmd.log(json[json.length - 1]);
+
+            if (this.verbose) {
+              cmd.log(vorpal.chalk.green('DONE'));
+            }
+          }
+          cb();
+      }, (err: any): void => this.handleRejectedPromise(err, cmd, cb));
+    };
+
+    if (args.options.confirm) {
+      removeLibrary();
+    }
+    else {
+      cmd.prompt({
+        type: 'confirm',
+        name: 'continue',
+        default: false,
+        message: `Are you sure you want to remove the library ${args.options.libraryUrl} as a central location for organization assets?`,
+      }, (result: { continue: boolean }): void => {
+        if (!result.continue) {
+          cb();
+        }
+        else {
+          removeLibrary();
+        }
+      });
+    }
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (!args.options.libraryUrl) {
+        return `Required parameter libraryUrl missing`;
+      }
+
+      return true
+    };
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '--libraryUrl <libraryUrl>',
+        description: 'The server relative URL of the library to be removed as a central location for organization assets'
+      },
+      {
+        option: '--confirm',
+        description: 'Don\'t prompt for confirming removing the organization asset library'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public commandHelp(args: CommandArgs, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(commands.ORGASSETSLIBRARY_REMOVE).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} to use this command you have to have permissions to access
+    the tenant admin site.
+
+  Examples:
+
+    Removes organization assets library without confirmation
+      ${commands.ORGASSETSLIBRARY_REMOVE} --libraryUrl /sites/branding/assets --confirm
```

Comment:
I added quotes to the path like `"/sites/branding/assets"`. I did that because users might have something like `/sites/branding/Shared Documents` set as org assets and if they do not pay attention and blindly follow the example. Then they can come back to us with funny question :)